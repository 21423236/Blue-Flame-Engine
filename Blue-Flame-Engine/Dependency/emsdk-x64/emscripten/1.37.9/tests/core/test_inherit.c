version https://git-lfs.github.com/spec/v1
oid sha256:4213842809aac2f951306d132e504a04e4d40646bd055d36b5aac4f217259f88
size 363
