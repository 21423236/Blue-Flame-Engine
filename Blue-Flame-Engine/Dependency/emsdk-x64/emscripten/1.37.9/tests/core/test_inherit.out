version https://git-lfs.github.com/spec/v1
oid sha256:1c623a31cc61d466dfc4e05eda43a7cebcf97349b82b921ebb5ff213c674c329
size 25
